tests/build/build_templates.go

```diff
@@ -280,23 +281,28 @@ var _ = framework.BuildSuiteDescribe("Build templates E2E test", Label("build",
 						Sources: policySource,
 						Configuration: &ecp.EnterpriseContractPolicyConfiguration{
 							// The BuildahDemo pipeline used to generate the test data does not
-							// include the required test tasks, so this policy should always fail.
+							// include the required test tasks, so this policy should always fail.generator
```

Comment:
Not sure if this diff hunk is intentional..?
```suggestion
							// include the required test tasks, so this policy should always fail.
```